curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-string-and-array-methods-by-building-a-music-player/65420dcfc60580678dad7a92.md

```diff
@@ -25,16 +25,10 @@ You should create an `li` tag inside the backticks.
 assert.match(code, /return\s*`\s*<li\s*/)
 ```
 
-Your opening `li` tag should have the `id` `song-${song.id}` as its first attribute and value.
+Your opening `li` tag should have two attributes: an `id` attribute `song-${song.id}`, and a `class` attribute `playlist-song`.
```

Comment:
We should still have two tests here.
One test should be for the id attribute and one test should be for the class

You can look at earlier PR's that tackled the same issue
https://github.com/freeCodeCamp/freeCodeCamp/pull/53027/files

Reply:
The test cases have been updated, please see the screenshots below.

for the `id` attribute:

<img width="689" alt="Screenshot 2024-02-14 at 16 54 38" src="https://github.com/freeCodeCamp/freeCodeCamp/assets/40338102/fae5df52-6cf8-4b24-b7f3-6dc61122031b">

for the `class` attribute:

<img width="689" alt="Screenshot 2024-02-14 at 16 54 29" src="https://github.com/freeCodeCamp/freeCodeCamp/assets/40338102/ce363588-3f2e-4282-8763-911e0ff3a6a6">